### PR TITLE
[tw] Add content/zh-tw/function-as-a-service.md

### LINKED ISSUE #2229: [tw] Localize `Function as a Service (FaaS)` for Traditional Chinese
### Please describe why this term needs to be updated

- Localize `Function as a Service (FaaS)` for Traditional Chinese
- https://glossary.cncf.io/function-as-a-service/

### PR DESCRIPTION
**Pull request by kaiyee0**
### Describe your changes
Origin content from https://glossary.cncf.io/function-as-a-service/

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2229 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/function-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: 功能即服務 (FaaS)
```
Comment (pichuang) [inline content/zh-tw/function-as-a-service.md:2]: Personally, I think "函數即服務" is more common?

File: content/zh-tw/function-as-a-service.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+status: Completed
+category: 技術
+tags: ["基礎架構", "", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+功能即服務 (FaaS) 是一種[無伺服器](/serverless/)的[雲端運算](/cloud-computing/)[服務](/service/)，
```
Comment (pichuang) [inline content/zh-tw/function-as-a-service.md:10]:
```suggestion
功能即服務 (FaaS) 是一種[無伺服器](/zh-tw/serverless/)的[雲端運算](/zh-tw/cloud-computing/)[服務](/zh-tw/service/)，
```

File: content/zh-tw/function-as-a-service.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+它可以在特定事件觸發時執行程式碼，
+而無需去維護常見於建置與發佈[微服務](/microservices/)應用程式所需的複雜基礎設施。
```
Comment (pichuang) [inline content/zh-tw/function-as-a-service.md:12]:
```suggestion
而無需去維護常見於建置與發佈[微服務](/zh-tw/microservices/)應用程式所需的複雜基礎設施。
```

File: content/zh-tw/function-as-a-service.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+在 FaaS 的運作模式下，使用者只需專注於管理自己的功能和資料，應用程式的其他管理工作則交由雲端服務提供者來處理。
+這種服務不僅讓開發者可以取得他們所需的功能以外，而且在程式碼不執行時，也無需支付額外的服務費用。
+
+## 解決的問題 {#problem-it-addresses}
+
+在傳統的地端環境中，企業需要自行管理和維護其資料中心。
+企業必須投資於伺服器、儲存空間、軟體和其他技術外，
+可能需要聘請IT人員或是外包商來購買、管理並升級所有的設備和使用授權。
```
Comment (pichuang) [inline content/zh-tw/function-as-a-service.md:20]:
```suggestion
可能需要聘請 IT 人員或是外包商來購買、管理並升級所有的設備和使用授權。
```

File: content/zh-tw/function-as-a-service.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+在這種情況下，資料中心必須配置足夠的資源以應對高峰需求，即使在工作量減少、資源閒置時也難以避免。
+反觀，若企業業務快速擴展，IT 部門可能又面臨資源無法即時提供的困境。
+在標準的[基礎設施即服務 (IaaS)](/infrastructure-as-a-service/) 雲端運算模型下，
```
Comment (pichuang) [inline content/zh-tw/function-as-a-service.md:23]:
```suggestion
在標準的[基礎設施即服務 (IaaS)](/zh-tw/infrastructure-as-a-service/) 雲端運算模型下，
```

File: content/zh-tw/function-as-a-service.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+用戶會預先購買特定容量，即表示你需要為保持應用程式運行所需的伺服器元件支付給公共雲端服務提供商。
+在需求增大時，用戶必須擴充伺服器容量；需求降低時，則需降低容量。
+即使應用程式並未在被使用，但運行該應用程式所需的雲端基礎設施仍要持續運作。
+
+## 如何幫助 {#how-it-helps}
+
+FaaS 為開發者提供了一種[抽象](/abstraction/)，使他們能夠在無需管理伺服器的情況下，可以根據事件來運行網路應用程式。
```
Comment (pichuang) [inline content/zh-tw/function-as-a-service.md:30]:
```suggestion
FaaS 為開發者提供了一種[抽象](/zh-tw/abstraction/)，使他們能夠在無需管理伺服器的情況下，可以根據事件來運行網路應用程式。
```

File: content/zh-tw/function-as-a-service.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+以檔案上傳為例，這個動作可能會觸發一段自定義的程式碼，該程式碼將原始檔案轉換成多種格式。
+對於大量使用的程式碼，FaaS 的基礎架構能自動擴縮，
+無需開發者花費時間和資源去設計程式碼的[可擴展性](/scalability/)。
```
Comment (pichuang) [inline content/zh-tw/function-as-a-service.md:33]:
```suggestion
無需開發者花費時間和資源去設計程式碼的[可擴展性](/zh-tw//scalability/)。
```

**Review by pichuang — COMMENTED** [review]
Thanks for your contribution. I put my comment inline

File: content/zh-tw/function-as-a-service.md (hunk 1 above)
Comment (kaiyee0) [reply content/zh-tw/function-as-a-service.md:2]: 我原本也是這樣想，但是我看 wiki 已經有人優先翻譯成『功能』了
https://zh.wikipedia.org/zh-tw/%E5%8A%9F%E8%83%BD%E5%8D%B3%E6%9C%8D%E5%8B%99
What do you think

File: content/zh-tw/function-as-a-service.md (hunk 2 above)
Comment (kaiyee0) [reply content/zh-tw/function-as-a-service.md]: 因為頁面不存在，所以會導致 not found，所以我才先用英文的頁面。
已修正。

File: content/zh-tw/function-as-a-service.md (hunk 4 above)
Comment (kaiyee0) [reply content/zh-tw/function-as-a-service.md]: Fix

File: content/zh-tw/function-as-a-service.md (hunk 7 above)
Comment (kaiyee0) [reply content/zh-tw/function-as-a-service.md]: Fix

File: content/zh-tw/function-as-a-service.md (hunk 1 above)
Comment (ydFu) [reply content/zh-tw/function-as-a-service.md:2]: Based on the data obtained from the reference cloud service providers, while both translations(函式即服務,功能即服務) are utilized by vendors, there is a noticeably higher frequency of usage for one particular term(函式即服務).

函式即服務 (FaaS):

1. Google Cloud:
https://cloud.google.com/functions?hl=zh-tw

2. Oracle:
https://www.oracle.com/tw/cloud/cloud-native/functions/what-is-serverless/

3. microsoft Azure
https://learn.microsoft.com/zh-tw/azure/architecture/web-apps/serverless/architectures/code

4. F5
https://www.f5points.com.tw/tag/faas/

5. IBM Cloud
https://cloud.ibm.com/docs/openwhisk?topic=openwhisk-getting-started&locale=zh-TW

6. AWS Amazon
https://aws.amazon.com/tw/serverless/faqs/

功能即服務(FaaS):

1. Intel:
https://www.intel.com.tw/content/www/tw/zh/cloud-computing/faas.html

2. TIBCO
https://www.tibco.com/zh-hant/reference-center/what-is-function-as-a-service-faas

3. RedHat
https://www.redhat.com/zh-tw/topics/cloud

4. WIKI
https://zh.wikipedia.org/zh-tw/%E5%8A%9F%E8%83%BD%E5%8D%B3%E6%9C%8D%E5%8B%99

Comment (kaiyee0) [reply content/zh-tw/function-as-a-service.md:2]: 感謝補充，已修正

File: content/zh-tw/function-as-a-service.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+status: Completed
+category: 技術
+tags: ["基礎架構", "", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+函式即服務 (FaaS) 是一種[無伺服器](/zh-tw/serverless/)的[雲端運算](/zh-tw/cloud-computing/)[服務](/zh-tw/service/)，
+它可以在特定事件觸發時執行程式碼，
+而無需去維護常見於建置與發佈[微服務](/zh-tw/microservices/)應用程式所需的複雜基礎設施。
```
Comment (ydFu) [inline content/zh-tw/function-as-a-service.md:12]:
```suggestion
而無需去維護常見於建置與發佈[微服務](/zh-tw/microservices/)應用程式所需的複雜基礎架構。
```
infrastructure -> 基礎架構

File: content/zh-tw/function-as-a-service.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+這種服務不僅讓開發者可以取得他們所需的功能以外，而且在程式碼不執行時，也無需支付額外的服務費用。
+
+## 解決的問題 {#problem-it-addresses}
+
+在傳統的地端環境中，企業需要自行管理和維護其資料中心。
+企業必須投資於伺服器、儲存空間、軟體和其他技術外，
+可能需要聘請 IT 人員或是外包商來購買、管理並升級所有的設備和使用授權。
+在這種情況下，資料中心必須配置足夠的資源以應對高峰需求，即使在工作量減少、資源閒置時也難以避免。
+反觀，若企業業務快速擴展，IT 部門可能又面臨資源無法即時提供的困境。
+在標準的[基礎設施即服務 (IaaS)](/zh-tw/infrastructure-as-a-service/) 雲端運算模型下，
```
Comment (ydFu) [inline content/zh-tw/function-as-a-service.md:23]:
```suggestion
在標準的[基礎架構即服務 (IaaS)](/zh-tw/infrastructure-as-a-service/) 雲端運算模型下，
```
infrastructure -> 基礎架構

File: content/zh-tw/function-as-a-service.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+用戶會預先購買特定容量，即表示你需要為保持應用程式運行所需的伺服器元件支付給公共雲端服務提供商。
+在需求增大時，用戶必須擴充伺服器容量；需求降低時，則需降低容量。
+即使應用程式並未在被使用，但運行該應用程式所需的雲端基礎設施仍要持續運作。
```
Comment (ydFu) [inline content/zh-tw/function-as-a-service.md:26]:
```suggestion
即使應用程式並未在被使用，但運行該應用程式所需的雲端基礎架構仍要持續運作。
```
infrastructure -> 基礎架構

File: content/zh-tw/function-as-a-service.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+category: 技術
+tags: ["基礎架構", "", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+函式即服務 (FaaS) 是一種[無伺服器](/zh-tw/serverless/)的[雲端運算](/zh-tw/cloud-computing/)[服務](/zh-tw/service/)，
+它可以在特定事件觸發時執行程式碼，
+而無需去維護常見於建置與發佈[微服務](/zh-tw/microservices/)應用程式所需的複雜基礎設施。
+在 FaaS 的運作模式下，使用者只需專注於管理自己的功能和資料，應用程式的其他管理工作則交由雲端服務提供者來處理。
```
Comment (ydFu) [inline content/zh-tw/function-as-a-service.md:13]:
```suggestion
在 FaaS 的運作模式下，使用者只需專注於管理自己的功能和資料，應用程式的其他管理工作則交由雲端服務提供商來處理。
```
 cloud provider ->  雲端服務提供商

**Review by ydFu — COMMENTED** [review]
It is recommended to maintain consistency in translation within the same article to avoid presenting two different Chinese translations for a single English word. Consistency is crucial in delivering a clear and cohesive message to the target audience. Employing a uniform translation approach throughout the article will enhance its overall readability and comprehension.

File: content/zh-tw/function-as-a-service.md (hunk 8 above)
Comment (kaiyee0) [reply content/zh-tw/function-as-a-service.md:12]: fix

File: content/zh-tw/function-as-a-service.md (hunk 9 above)
Comment (kaiyee0) [reply content/zh-tw/function-as-a-service.md:23]: fix

File: content/zh-tw/function-as-a-service.md (hunk 10 above)
Comment (kaiyee0) [reply content/zh-tw/function-as-a-service.md:26]: fix

**Comment by kaiyee0** [comment]
Thanks @ydFu , the article is now updated.

**Review by ydFu — APPROVED** [review]
LGTM, thank @kaiyee0

**Review by pichuang — APPROVED** [review]
LGTM. Thanks for your first contribution.

**Pull request merged**